custom_model_runner/datarobot_drum/drum/args_parser.py

```diff
@@ -1186,6 +1282,18 @@ def verify_options(options):
         CMRunnerArgsRegistry.verify_dr_api_access_options(options, options.subparser_name)
         CMRunnerArgsRegistry.verify_triton_server_options(options, options.subparser_name)
 
+        # Add validation for time series args requiring use_datarobot_predict
+        time_series_args = ["forecast_point", "predictions_start_date", "predictions_end_date"]
+        if any(getattr(options, arg, None) is not None for arg in time_series_args):
+            if options.subparser_name != ArgumentsOptions.SCORE:
+                print("\nError: Time series arguments can only be used with the score command")
+                exit(1)
+            if not getattr(options, "use_datarobot_predict", False):
+                print(
+                    "\nError: Time series arguments require --use-datarobot-predict to be enabled"
+                )
+                exit(1)
```

Comment:
Can this get moved into a helper function: `CMRunnerArgsRegistry.verify_timeseries_args(...)